determinism_tex.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
This was added in your first commit. You'll need to rebase and edit the first commit to remove the file.